.rubocop.yml

```diff
@@ -1,74 +1,520 @@
 ---
-inherit_from: './.rubocop_todo.yml'
+require:
+- rubocop-performance
+- rubocop-rspec
 AllCops:
-    TargetRubyVersion: 1.9
-
-Style/HashSyntax:
-    EnforcedStyle: hash_rockets
-
-Layout/FirstArrayElementLineBreak:
-    Enabled: true
-Layout/FirstHashElementLineBreak:
-    Enabled: true
-Layout/FirstMethodArgumentLineBreak:
-    Enabled: true
-Layout/FirstMethodParameterLineBreak:
-    Enabled: true
-Layout/IndentArray:
-    EnforcedStyle: consistent
-Layout/MultilineArrayBraceLayout:
-    EnforcedStyle: new_line
-Layout/MultilineAssignmentLayout:
-    Enabled: true
-    EnforcedStyle: same_line
-Layout/MultilineHashBraceLayout:
-    EnforcedStyle: new_line
-Layout/MultilineMethodDefinitionBraceLayout:
-    EnforcedStyle: new_line
-
-Style/AutoResourceCleanup:
-    Enabled: true
+  DisplayCopNames: true
+  TargetRubyVersion: '2.5'
```

Comment:
Given that Ruby 2.5 is EoL, should we drop support for that as well and. go strait to 2.6 or 2.7?

Reply:
I'm very tempted. I really don't like supporting EOL things.

Let me discuss internally tomorrow to collect some more views.

Reply:
Go for it